websites/B/Behance/presence.ts

```diff
@@ -133,10 +133,11 @@ presence.on("UpdateData", async () => {
 			// Intentional `delete` for pause boolean
 			delete presenceData.startTimestamp;
 			if (!video.paused) {
-				presenceData.endTimestamp =
-					Date.now() / 1000 +
-					presence.timestampFromFormat(video.duration) -
-					presence.timestampFromFormat(video.currentTime);
+				[presenceData.startTimestamp, presenceData.endTimestamp] =
+					presence.getTimestamps(
+						Math.floor(video.current),
+						Math.floor(video.duration)
```

Comment:
Video time stamps are retrieved as text from html elements. Review interface and data when changing this presence.